web_accounts_list.json

```diff
@@ -4436,6 +4436,72 @@
           "known_accounts" : ["tata23", "fitzsta"],
           "category" : "XXXPORNXXX",
           "valid" : true
-        }
+        },
+        {
+          "name" : "Snapchat",
+          "check_uri" : "https://story.snapchat.com/s/{account}",
+          "account_existence_code" : "200",
+          "account_existence_string" : "is on Snapchat!",
+          "account_missing_string" : "Not_Found",
+          "account_missing_code" : "404",
+          "known_accounts" : ["djkhaled305", "mileycyrus"],
+          "category" : "social",
+          "valid" : true
+        },
+        {
+          "name" : "Spotify",
+          "check_uri" : "https://open.spotify.com/user/{account}",
+          "account_existence_code" : "200",
+          "account_existence_string" : "on Spotify</title>",
+          "account_missing_string" : "Spotify - Web Player</title>",
+          "account_missing_code" : "200",
+          "known_accounts" : ["kexp_official", "mkbhd"],
+          "category" : "music",
+          "valid" : true
+        }, 
+        {
+          "name" : "ArtBreeder",
+          "check_uri" : "https://www.artbreeder.com/{account}",
+          "account_existence_code" : "200",
+          "account_existence_string" : "<title>",
+          "account_missing_string" : "Not found:",
+          "account_missing_code" : "404",
+          "known_accounts" : ["dolores", "cyborghyena"],
+          "category" : "art",
+          "valid" : true
+          },
+          {
+           "name" : "BDSMLR",
+           "check_uri" : "https://{account}.bdsmlr.com",
+           "account_existence_code" : "200",
+           "account_existence_string" : "This blog contains adult content",
+           "account_missing_string" : "This blog doesn't exist.",
+           "account_missing_code" : "200",
+           "known_accounts" : ["themunch", "coffeemeandporn"],

```

Comment:
Can you pick a different account instead of "coffeemeandporn" as that one now requires authentication to view.

Reply:
Changed to "shibari4all" which doesn't require auth and modified the detect string to work for accounts/blogs that are public and ones that require auth.